entity-framework/core/modeling/relationships/many-to-many.md

```diff
@@ -0,0 +1,1071 @@
+
+<!--
+            protected override void OnModelCreating(ModelBuilder modelBuilder)
+            {
+                modelBuilder.Entity<Post>()
+                    .HasMany(e => e.Tags)
+                    .WithMany(e => e.Posts)
+                    .UsingEntity(
+                        "PostTag",
+                        l => l.HasOne(typeof(Tag)).WithMany().HasForeignKey("TagsId").HasPrincipalKey(nameof(Tag.Id)),
+                        r => r.HasOne(typeof(Post)).WithMany().HasForeignKey("PostsId").HasPrincipalKey(nameof(Post.Id)),
+                        j => j.HasKey("PostsId", "TagsId"));
+            }
+-->
+[!code-csharp[BasicManyToManyFullConfig](../../../../samples/core/Modeling/Relationships/ManyToMany.cs?name=BasicManyToManyFullConfig)]
+
+> [!IMPORTANT]
+> Please don't attempt to fully configure everything even when it is not needed. As can be seen above, the code gets complicated quickly and its easy to make a mistake. And even in the example above there are many things in the model that are still configured by convention. It's not realistic to think that everything in an EF model can always be fully configured explicitly.
+
+Regardless of whether the relationship is built by convention or using either of the shown explicit configurations, the resulting mapped schema (using SQLite) is:
+
+```sql
+CREATE TABLE "Posts" (
+    "Id" INTEGER NOT NULL CONSTRAINT "PK_Posts" PRIMARY KEY AUTOINCREMENT);
+
+CREATE TABLE "Tags" (
+    "Id" INTEGER NOT NULL CONSTRAINT "PK_Tags" PRIMARY KEY AUTOINCREMENT);
+
+CREATE TABLE "PostTag" (
+    "PostsId" INTEGER NOT NULL,
+    "TagsId" INTEGER NOT NULL,
+    CONSTRAINT "PK_PostTag" PRIMARY KEY ("PostsId", "TagsId"),
+    CONSTRAINT "FK_PostTag_Posts_PostsId" FOREIGN KEY ("PostsId") REFERENCES "Posts" ("Id") ON DELETE CASCADE,
+    CONSTRAINT "FK_PostTag_Tags_TagsId" FOREIGN KEY ("TagsId") REFERENCES "Tags" ("Id") ON DELETE CASCADE);
+```
+
+> [!TIP]
+> When using a Database First flow to [scaffold a DbContext from an existing database](xref:core/managing-schemas/scaffolding), EF Core 6 and later looks for this pattern in the database schema and scaffolds a many-to-many relationship as described in this document. This behavior can be changed through use of a [custom T4 template](xref:core/managing-schemas/scaffolding/templates). For other options, see [_Many-to-many relationships without mapped join entities are now scaffolded_](xref:core/what-is-new/ef-core-6.0/breaking-changes#many-to-many).
+
+> [!IMPORTANT]
+> Currently, EF Core uses `Dictionary<string, object>` to represent join entity instances for which no .NET class has been configured. However, to improve performance, a different type may be used in a future EF Core release. Do not depend on the join type being `Dictionary<string, object>` unless this has been explicitly configured.
+
+### Unidirectional many-to-many
```

Comment:
Consider putting this further down - it seems rather niched compared to e.g. the named join table example.